samples/sample.rc

```diff
@@ -17,6 +17,8 @@
 // starting with "a"
 aaaaaaaa ICON "sample.ico"
 
+horse    CURSOR "horse.cur"
+
```

Comment:
We really don't want to add it here. Please add it to the test .rc file.

Reply:
@vadz 
As mentioned before, there is no such file:

```
Igor@fordanwork MINGW64 /c/wxFork (GetCursorFromBundleResoure_part2)
$ find . -name *\.rc
./demos/bombs/bombs.rc
./demos/forty/forty.rc
./demos/fractal/fractal.rc
./demos/life/life.rc
./demos/poem/wxpoem.rc
./include/wx/msw/wx.rc
./samples/dnd/dnd.rc
./samples/docview/docview.rc
./samples/dragimag/dragimag.rc
./samples/image/image.rc
./samples/listctrl/listtest.rc
./samples/mdi/mdi.rc
./samples/mfc/mfctest.rc
./samples/nativdlg/dialog1.rc
./samples/nativdlg/nativdlg.rc
./samples/ownerdrw/ownerdrw.rc
./samples/regtest/regtest.rc
./samples/sample.rc
./samples/sashtest/sashtest.rc
./samples/sound/sound.rc
./samples/svg/svgtest.rc
./samples/toolbar/toolbar.rc
./src/expat/expat/win32/version.rc
./src/msw/version.rc
./src/png/contrib/visupng/VisualPng.rc
./src/png/scripts/pngwin.rc
./src/zlib/contrib/vstudio/vc10/zlib.rc
./src/zlib/contrib/vstudio/vc11/zlib.rc
./src/zlib/contrib/vstudio/vc12/zlib.rc
./src/zlib/contrib/vstudio/vc14/zlib.rc
./src/zlib/contrib/vstudio/vc9/zlib.rc
./src/zlib/win32/zlib1.rc
./utils/helpview/src/client.rc
./utils/helpview/src/helpview.rc
./utils/hhp2cached/hhp2cached.rc
./utils/screenshotgen/src/screenshotgen.rc
./utils/wxrc/wxrc.rc
```

I can either create one or copy the horse.cur file over to sample directory.

Thank you.

Reply:
Yes, you obviously need to add a `test.rc` file if you want to use the resources in the test. I'm sorry but how exactly do you think it's going to work otherwise?